stdlib/Logging/docs/src/index.md

```diff
@@ -193,6 +193,24 @@ Similarly, the environment variable can be used to enable debug logging of
 modules, such as `Pkg`, or module roots (see [`Base.moduleroot`](@ref)). To
 enable all debug logging, use the special value `all`.
 
+To turn debug logging on from the REPL, set `ENV["JULIA_DEBUG"]` to the
+name of the module of interest. Functions defined in the REPL belong to 
```

Comment:
This trailing whitespace broke CI on master.  https://github.com/JuliaLang/julia/pull/33641

Reply:
Oh dear, looks like this was my mistake.